pkgs/tools/package-management/packagekit/default.nix

```diff
@@ -1,52 +1,82 @@
-{ stdenv, fetchFromGitHub, lib
-, intltool, glib, pkg-config, polkit, python3, sqlite
-, gobject-introspection, vala, gtk-doc, autoreconfHook, autoconf-archive
-, nix, enableNixBackend ? false, boost
+{ stdenv
+, fetchFromGitHub
+, lib
+, intltool
+, glib
+, pkg-config
+, polkit
+, python3
+, sqlite
+, gobject-introspection
+, vala
+, gtk-doc
+, nix
+, boost
+, meson
+, ninja
+, nlohmann_json
+, libxslt
+, libxml2
+, gst_all_1
+, gtk3
 , enableCommandNotFound ? false
-, enableBashCompletion ? false, bash-completion ? null
-, enableSystemd ? stdenv.isLinux, systemd }:
+, enableBashCompletion ? false
+, bash-completion ? null
+, enableSystemd ? stdenv.isLinux
+, systemd
+}:
 
 stdenv.mkDerivation rec {
   pname = "packagekit";
-  version = "1.1.13";
+  version = "1.2.5pre";
 
   outputs = [ "out" "dev" ];
 
   src = fetchFromGitHub {
     owner = "hughsie";
     repo = "PackageKit";
-    rev = "PACKAGEKIT_${lib.replaceStrings ["."] ["_"] version}";
-    sha256 = "0xmgac27p5z8wr56yw3cqhywnlvaf8kvyv1g0nzxnq167xj5vxam";
+    rev = "e8140563ec7f78dd723068396dcad40e47580379";
+    sha256 = "0mz54km3fy1rlbldz2nj778022i84gvvy6hyk2jym815qlvr3mkq";
   };
 
-  buildInputs = [ glib polkit python3 gobject-introspection ]
-                  ++ lib.optional enableSystemd systemd
-                  ++ lib.optional enableBashCompletion bash-completion;
-  propagatedBuildInputs =
-    [ sqlite boost ]
-    ++ lib.optional enableNixBackend nix;
-  nativeBuildInputs = [ vala intltool pkg-config autoreconfHook autoconf-archive gtk-doc ];
-
-  preAutoreconf = ''
-    gtkdocize
-    intltoolize
-  '';
+  buildInputs = [
+    glib
+    polkit
+    python3
+    gobject-introspection
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gtk3
+    nlohmann_json
+  ] ++ lib.optional enableSystemd systemd
+  ++ lib.optional enableBashCompletion bash-completion;
+  propagatedBuildInputs = [
+    sqlite
+    boost
+    nix
+  ];
+  nativeBuildInputs = [
+    vala
+    intltool
+    pkg-config
+    gtk-doc
```

Comment:
`gtk_doc` is off by default.

Reply:
But we can trivially enable it, along with man pages:

```patch
--- a/pkgs/tools/package-management/packagekit/default.nix
+++ b/pkgs/tools/package-management/packagekit/default.nix
@@ -16,6 +16,8 @@
 , ninja
 , nlohmann_json
 , libxslt
+, docbook-xsl-nons
+, docbook_xml_dtd_42
 , libxml2
 , gst_all_1
 , gtk3
@@ -30,7 +32,7 @@ stdenv.mkDerivation rec {
   pname = "packagekit";
   version = "1.2.5pre";
 
-  outputs = [ "out" "dev" ];
+  outputs = [ "out" "dev" "devdoc" ];
 
   src = fetchFromGitHub {
     owner = "hughsie";
@@ -62,6 +64,8 @@ stdenv.mkDerivation rec {
     gtk-doc
     meson
     libxslt
+    docbook-xsl-nons
+    docbook_xml_dtd_42
     libxml2
     ninja
   ];
@@ -73,7 +77,7 @@ stdenv.mkDerivation rec {
     "-Ddbus_services=${placeholder "out"}/share/dbus-1/system-services"
     "-Dsystemdsystemunitdir=${placeholder "out"}/lib/systemd/system"
     "-Dcron=false"
-    "-Dman_pages=false"
+    "-Dgtk_doc=true"
   ]
   ++ lib.optional (!enableBashCompletion) "-Dbash_completion=false"
   ++ lib.optional (!enableCommandNotFound) "-Dbash_command_not_found=false";
```

Reply:
thanks! that works